setup/TemplateConfigurator.rb

```diff
@@ -188,7 +188,13 @@ def validate_user_details
     #----------------------------------------#
 
     def user_name
-      (ENV['GIT_COMMITTER_NAME'] || `git config user.name`).strip
+      (ENV['GIT_COMMITTER_NAME'] || github_user_name || `git config user.name`).strip
```

Comment:
If you make this fall back to `<GITHUB_USERNAME>` at the end, then this is good to go